docs/api/version-history.md

```diff
@@ -38,6 +38,8 @@ keywords: "API, Docker, rcli, REST, documentation"
   defined through `Mounts` because the `VolumeDriver` option has no effect on
   those volumes. This warning was previously generated by the CLI, but now
   moved to the daemon so that other clients can also get this warning.
+* `POST /containers/create` now supports `Mount` of type `Image` for mounting
```

Comment:
nit: on the API level the string is lowercase
https://github.com/moby/moby/blob/1d12ad0672ea19bce120d2727e186632b122b335/api/types/mount/mount.go#L22-L23

```suggestion
* `POST /containers/create` now supports `Mount` of type `image` for mounting
```

Reply:
Done!